deprecate.dd

```diff
@@ -55,6 +56,48 @@ $(SPEC_S Deprecated Features,
     )
 
 
+$(H3 $(DEPNAME Using the result of a comma expression))
+    $(P The comma operator (`,`) allows executing multiple expressions and
+        discards the result of them except for the last which is returned.
+        ---
+        int a = 1;
+        int b = 2;
+        bool ret = a == 2, b == 2; // true
+        ---
+
+        It's also common to use the comma operator in for-loop increment
+        statements to allow multiple expressions.
+
+        ---
+        for (; !a.empty && !b.empty; a.popFront, b.popFront)
+        ---
+    )
+$(H4 Corrective Action)
+    $(P Using the result of the comma operator should be forbidden.
```

Comment:
This should be a description of how to fix code - ie replace with a lambda or statement.  Check the other items on this page for other examples of what the Corrective Action section is for.